Added social media component

awinny opened the pull request:
**Describe pull-request**</br>
- Created a social media component
- Added example of social media items inside the footer story

awinny committed:
Added social media component

awinny committed:
Merge branch 'master' into feature/Add_social_media_component

# Conflicts:
#	.storybook/components/c-footer.js
#	src/components/footer/footer.tsx
#	themes/scania/c-footer.scss

awinny committed:
Added support for target to social media
- Added support for target on scoial media items

helloimela reviewed (changes requested):
- Styling in mobile view 
- Should add the real social media brands icon on this pull request

awinny committed:
Merge branch 'master' into feature/Add_social_media_component

awinny committed:
Added missing mobile styling + improvements
- Added missing mobile styling
- Updated header and footer stories to reflect on the new changes
- Changed social media setup in the footer to be more logic

awinny committed:
Changed social media items
- Changed icons in footer examples to represent real social media items
- Added brands as part of our icons lib
